apps/spruce/src/pages/projectSettings/tabs/GithubPermissionGroupsTab/FieldTemplates.tsx

```diff
@@ -0,0 +1,107 @@
+import { useRef } from "react";
+import styled from "@emotion/styled";
+import Button from "@leafygreen-ui/button";
+import { useLeafyGreenTable, LGColumnDef } from "@leafygreen-ui/table";
+import { Body } from "@leafygreen-ui/typography";
+import { ArrayFieldTemplateProps, ObjectFieldTemplateProps } from "@rjsf/core";
+import { PlusButton } from "components/Buttons";
+import Icon from "components/Icon";
+import { getFields } from "components/SpruceForm/utils";
+import { BaseTable } from "components/Table/BaseTable";
+import { size, tableColumnOffset } from "constants/tokens";
+import { Unpacked } from "types/utils";
+
+export const PermissionObjectFieldTemplate: React.FC<
+  Pick<ObjectFieldTemplateProps, "formData" | "properties">
+> = ({ formData, properties }) => {
+  const [permissionType, permissionValue] = getFields(
+    properties,
+    formData.isDisabled,
+  );
+  return (
+    <RowContainer>
+      {permissionType}
+      {permissionValue}
+    </RowContainer>
```

Comment:
Should we validate against duplicate keys or key/value pairs in the table?
<img width="582" alt="image" src="https://github.com/user-attachments/assets/2f4439f9-18f6-47f2-bba9-30996187e174">

Reply:
added validation but would note that the error message messes up the alignment with the trash can icon, but it's not really fixable ☹️